File: MuySamples/app/src/main/java/com/muy/muysamples/widget/WidgetSampleActivity.java
package com.muy.muysamples.widget;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.view.ViewGroup;

import com.muy.muysamples.R;

public class WidgetSampleActivity extends AppCompatActivity {

    private BottomPopup popup;

    private ViewGroup root;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_checkbox_sample);

        root = findViewById(R.id.root);

        popup = new BottomPopup(this);

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                popup.show(root);
            }
        });
    }
}

File: MuySamples/app/src/main/java/com/muy/muysamples/widget/WidgetSampleActivity.kt
package com.muy.muysamples.widget

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.muy.muysamples.R

class WidgetSampleActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.cutom_view)
//        setContentView(R.layout.placeholder)
//        iv_app_icon.setOnClickListener { v -> placeholder.setContentId(v.id) }
    }
}